LICENSE

```diff
@@ -1,4 +1,21 @@
+MIT License
 
-Copyright (c) 2018 Ryan Stevens
+Copyright (c) 2018 Ryan Stevens and Josh Everett. All rights reserved.
```

Comment:
I say we just keep it like this until we need to change the copyright.  Happy to discuss "... and contributors" when there are contributors.  

cc @josheverett